.github/workflows/build.yaml

```
@@ -34,8 +34,8 @@ jobs:
       run: |
         git clone --depth 1 http://github.com/BinaryAnalysisPlatform/bap-frames.git
         git clone --depth 1 http://github.com/BinaryAnalysisPlatform/qemu.git
-    - name: Build without tracewrap
+    - name: Build for Hexagon
       run: |
         cd qemu
-        ./configure --prefix=$HOME --target-list=arm-linux-user
+        ./configure --prefix=$HOME --with-tracewrap="$HOME"/bap-frames --target-list=hexagon-linux-user
```

to make opam-installed stuff visible (i.e., to update the PATH variable and the rest) you can use `opam exec --`, e.g.,
```suggestion
        opam exec -- ./configure --prefix=$HOME --with-tracewrap="$HOME"/bap-frames --target-list=hexagon-linux-user
```